packages/core/src/app/shipping/StaticConsignment.scss

```diff
@@ -1,5 +1,9 @@
 @import '../ui/Base';
 
+.staticConsignment {
```

Comment:
Just want to check if this causes issues with existing components? I don't think so but just want to check.

Reply:
This element is display: block by default so it fills up 100% of parent block width. However, for custom themes some common css classes can have display flex on child blocks, so the staticConsignment changes its width to a size, that fits the content.